.eslintrc

```diff
@@ -16,7 +16,8 @@
     "flowtype/require-valid-file-annotation": "off",
     "import/named": "off",
     "no-console": "off",
-    "no-unused-expressions": "off"
+    "no-unused-expressions": "off",
+    "no-await-in-loop": "off"
```

Comment:
Just remove the rule from the lower level `packages/reputation-miner/.eslintrc` now we have it project wide here.

Reply:
👍